src/runtime-rs/crates/hypervisor/src/device/driver/virtio_net.rs

```diff
@@ -54,7 +54,8 @@ pub struct NetworkConfig {
 pub struct NetworkDevice {
     /// Unique identifier of the device
     pub device_id: String,
-
+    /// network device driver: virtio-net-pci,
+    pub device_driver: String,
```

Comment:
This is code common to all hypervisor drivers.  Is this field appropriate here, given that it's only set to what appears to be a qemu-specific value?

It seems to me that the qemu driver should do without this field and figure out the network device driver by itself, from other information.  What do you think?

Reply:
> This is code common to all hypervisor drivers. Is this field appropriate here, given that it's only set to what appears to be a qemu-specific value?
> 
> It seems to me that the qemu driver should do without this field and figure out the network device driver by itself, from other information. What do you think?
>
Fixed

Reply:
> > This is code common to all hypervisor drivers. Is this field appropriate here, given that it's only set to what appears to be a qemu-specific value?
> > It seems to me that the qemu driver should do without this field and figure out the network device driver by itself, from other information. What do you think?
> 
> I fully agree with you. This should be hidden in the QEMU driver for now.
> 
Thx Greg @gkurz I agree with you that the driver is needed to hidden in Qemu driver deduced with more info such as transport and net device type  or others.